sdk/batch/Microsoft.Azure.Batch/Tools/CodeGenerationLibrary/CodeGenerationLibrary.csproj

```diff
@@ -1,18 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>CodeGenerationLibrary</PackageId>
     <Description>Azure Batch object model code generation library</Description>
     <!--After fixing all warnings, enable this option-->
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <AssemblyName>CodeGenerationLibrary</AssemblyName>
+    <AssemblyTitle>CodeGenerationLibrary</AssemblyTitle>
```

Comment:
Why this change? I wouldn't expect you need AssemblyName or AssemblyTitle.

Reply:
Why would it not be needed? Is it because it is not a shipped package?

Reply:
It isn't needed for the same reason AssemblyName isn't needed because it is redundant and defaults from the AssemblyName which defaults from the Project Name, https://github.com/dotnet/sdk/blob/2eb6c546931b5bcb92cd3128b93932a980553ea1/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DefaultAssemblyInfo.targets#L23.

Reply:
Thanks